converter_app/readers/sec.py

```diff
@@ -0,0 +1,127 @@
+import logging
+import re
+
+from .base import Reader
+
+logger = logging.getLogger(__name__)
+
+UNIT_EXTENSION = "_unit"
+
+
+class SecReader(Reader):
+    identifier = 'sec_reader'
+    priority = 95
+
+    _has_header = False
+    _has_first_value = False
+    _is_table_empty = True
+    _is_calibration = 0
+
+    def check(self):
+        result = False
+        if self.file.suffix.lower() == '.txt' and self.file.mime_type == 'text/plain':
+            first_lines = [self.file.string.splitlines()[0], self.file.string.splitlines()[1], self.file.string.splitlines()[2]]
+            result = 'Sample :' in first_lines[0] and 'Method settings :' in first_lines[1] and 'Sequence table :' in first_lines[2]
+
+        logger.debug('result=%s', result)
+        return result
+
+    def _append_table(self, tables):
+        self._has_header = False
+        self._has_first_value = False
+        self._is_table_empty = True
+
+        return self.append_table(tables)
+
+    def _handle_line(self, line: str, table: dict[str, list | dict]):
```

Comment:
These type hints will seem to only work in Python >= 3.10 which is pretty new (for Python standards). Dropping Python <= 3.9 needs to be decided before merging this PR.

Reply:
Can be fixed with adding `from __future__ import annotations` at the very top of the file. No need for a Python 3.10 discussion.